tutorials/spanruler_restaurant_reviews/project.yml

```diff
@@ -26,21 +26,21 @@ description: |
 
   |          | NER only  | With Spanruler  |
   |----------|-----------|-----------------|
-  | Price    | 83.72     | **84.88**       |
-  | Rating   | 77.21     | **77.78**       |
-  | Hours    | 64.78     | **65.31**       |
-  | Amenity  | 66.67     | **67.61**       |
-  | Location | 81.17     | **81.99**       |
-  | Restaurant_Name| 76.28     | **78.44**       |
+  | Price    | 81.68     | **83.23**       |
+  | Rating   | 78.42     |   78.06         |
```

Comment:
The `Rating` rules are making it worse somehow.

Reply:
I'm going to look into this and see if it has to do with annotation inconsistencies!